CONTRIBUTING.md

```diff
@@ -18,11 +17,10 @@ will likely make it into a release a little quicker.
 
 5. Push to your fork and submit a pull request.
 
-
 ## Dependencies
 
-The testing and development tools have a bunch of dependencies,
-all managed by [Bundler](http://bundler.io/) according to the
+This module is developed using PDK, and all dependencies are managed by
+[Bundler](http://bundler.io/) provided through PDK aoccording to the
 [Puppet support matrix](http://docs.puppetlabs.com/guides/platforms.html#ruby-versions).
```

Comment:
How about removing L24-36 and replacing with a link to https://puppet.com/docs/pdk/1.x/pdk_testing.html ?